en/flight_modes/mission.md

```diff
@@ -58,26 +58,56 @@ For more information about mission planning, see:
 * [Mission Planning](../flying/missions.md)
 * [Plan View](https://docs.qgroundcontrol.com/master/en/PlanView/PlanView.html) (*QGroundControl* User Guide)
 
+## Mission Feasibility Checks

```

Comment:
@sfuhrer FYI, I moved the more detailed docs into mission mode as discussed.

I guess we wait on @Igor-Misic before we merge? Also , I assume that an unfeasible mission on arming will not be deleted?